clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/controllers/AmazonLoadBalancerController.groovy

```diff
@@ -42,8 +42,9 @@ class AmazonLoadBalancerController {
 
   @RequestMapping(method = RequestMethod.GET)
   List<AmazonLoadBalancerSummary> list() {
-    Collection<String> loadBalancers = cacheView.getIdentifiers(LOAD_BALANCERS.ns)
-    getSummaryForLoadBalancers(loadBalancers).values() as List
+    def searchKey = Keys.getLoadBalancerKey('*', '*', '*', null) + '*'
```

Comment:
Yup, it shouldn't exclude them for that reason. `**` matches either `:vpc`, or just the cluster name.